specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-06-01-preview/examples/confidentialWatchlists/GetConfidentialWatchlistItems.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "api-version": "2022-06-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "operationalInsightsResourceProvider": "Microsoft.OperationalInsights",
+    "watchlistAlias": "highValueAsset"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspaces/myWorkspace/providers/Microsoft.SecurityInsights/confidentialWatchlists/highValueAsset/WatchlistItems/fd37d325-7090-47fe-851a-5b5a00c3f576",
+            "name": "fd37d325-7090-47fe-851a-5b5a00c3f576",
+            "etag": "\"f2089bfa-0000-0d00-0000-601c58b42021\"",
+            "type": "Microsoft.SecurityInsights/confidentialWatchlists/WatchlistItems",
+            "properties": {
+              "watchlistItemType": "watchlist-item",
+              "watchlistItemId": "fd37d325-7090-47fe-851a-5b5a00c3f576",
```

Comment:
>watchlistItemId": "fd37d325-7090-47fe-851a-5b5a00c3f576", [](http://example.com/codeflow?start=15&length=57)

Wondering why do you need this property? Looks like it is same as the resource name?

Reply:
Correct. The watchlistItemId was introduced as alternative to name since the former is generated by the API (a Guid)